dashboard/settings.py

```diff
@@ -32,6 +32,10 @@
 # Google Analytics ID
 GA_ID = config('GA_ID', default='')
 
+# File values from .env
+CANVAS_FILE = config('CANVAS_FILE', default='0')
+LECCAP_FILE = config('LECCAP_FILE', default='1')
```

Comment:
What is the purpose of having these as environmental variables?

Reply:
@pushyamig yes. I think it is better to config from the settings file, so that we can generate the list dynamically via configuration.

Reply:
I think we're going to have to have a more complex settings file for this feature in the future. The change I mentioned Vincent too had the configuration loaded up from a JSON file. I don't think we'll be able to "easily" do all of the configuration via environment. 

I was even thinking of switching this project to use a JSON configuration rather than the environment so we can have more complex and secure configurations. We could load up the JSON in Openshift via a secret. I just haven't had time to switch over to that.

JSON is the way that configuration is done in Google App Engine.

Reply:
Mark this as resolved for now.